srcpkgs/imath/patches/split-pyimath-cmake-target.patch

```diff
@@ -1,39 +0,0 @@
---- a/config/ImathConfig.cmake.in
-+++ b/config/ImathConfig.cmake.in
-@@ -4,4 +4,7 @@
- @PACKAGE_INIT@
- 
- include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
-+if(EXISTS "${CMAKE_CURRENT_LIST_DIR}/PyImathTargets.cmake")
-+  include("${CMAKE_CURRENT_LIST_DIR}/PyImathTargets.cmake")
-+endif()
```

Comment:
This is the reason, PyImath needs to be split. Otherwise, `find_package(imath)` will not found if pyimath is not installed.

Reply:
So, no it's not ok to be removed